### DISCUSSION TITLE
Digital Ocean Droplet - Accessing database with PowerBi

### DISCUSSION THREAD

**Discussion by todenforce** [Budibase/budibase#6052] [Help] [answered]
Hi All
I have a need to connect PowerBi reporting tool to the Digital Ocean droplet database so I can perform some reports.
i have several tables in the Budibase database and not sure how I can connect PowerBi to this database?
Has anyone got any help on how to do this -I'm sure a few people have a need to connect the data to external reporting tools

TIA

**Comment by mike12345567** [comment, marked as answer]
Hi @todenforce - your best chance would be to use the Budibase REST API, you can use the row search endpoint to retrieve data from a particular table.

https://docs.budibase.com/reference/post_tables-tableid-rows-search

We don't use a SQL database within Budibase, we use a NoSQL database called CouchDB which could be difficult to integrate with, the API is likely to be a lot easier.
